package.json

```diff
@@ -64,7 +64,7 @@
     "eslint-config-prettier": "^6.6.0",
     "eslint-plugin-prettier": "^3.1.1",
     "eslint-plugin-react": "^7.16.0",
-    "optipng-bin": "^5.1.0",
+    "optipng-bin": "^6.0.0",
```

Comment:
https://github.com/imagemin/optipng-bin/releases/tag/v6.0.0

Reply:
Looking at the main readme on https://github.com/imagemin/optipng-bin, we should swap this one out for https://github.com/imagemin/imagemin-optipng rather than updating the version number